server/webdriver/tests/standalone/sdg_sanity.py

```diff
@@ -67,7 +67,7 @@ class Mode:
 
 OUTPUT_DIR = "output"
 
-COUNTRIES_JSON_FILE = "../../../../experimental/sdg-tracker/src/config/countries.json"
+COUNTRIES_JSON_FILE = "../../../../experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/config/countries.json"
```

Comment:
https://code.officialstatistics.org/undata2/data-commons/frontend-tester/-/blob/main/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/config/countries.json?ref_type=heads is the canonical path now?

Reply:
I don't know. We were using the previous location earlier but it seems to have moved to this one now. @dwnoble?

Reply:
That is the prod code base for SDG.

Reply:
OK. Loading from here now.

Reply:
the new path looks correct. We updated the path to match the UN folder structure